templates/member/navConnection.twig

```diff
@@ -0,0 +1,15 @@
+{% if connected %}
+    <li class="nav-item">
+        <a class="nav-link" href="/member-profile">Mon profil</a>
```

Comment:
set the `href` to `#` when implementing a new template. Just to be sure you can search them later easily

Reply:
Done.